Marlin/src/inc/Conditionals_LCD.h

```diff
@@ -141,6 +141,13 @@
   #define IS_RRD_SC 1
   #define IS_U8GLIB_SSD1306
 
+#elif ENABLED(SAV_3DGLCD)
+
+  #ifdef U8GLIB_SSD1306
+    #define IS_U8GLIB_SSD1306
```

Comment:
This define should only be set by `Condifionals_LCD.h` and it is mainly set for the benefit of `SanityCheck.h` to distinguish `U8GLIB_SSD1306` by itself versus a display that happens to also be `U8GLIB_SSD1306`.